benchmarks/benchmarks.py

```diff
@@ -0,0 +1,84 @@
+# Write the benchmarking functions here.
+# See "Writing benchmarks" in the asv docs for more information.
+import numpy as np
+import scipy.stats as st
+
+
+class Hist:
+	def time_histogram(self):
+		try:
+			data = np.random.rand(10000, 1000)
+			import numba
+
+			@numba.njit(cache=True)
+			def _hist(data):
+				return np.histogram(data, bins=100)
+			return _hist(data)
+		except ImportError:
+			data = np.random.rand(10000, 1000)
+			return np.histogram(data, bins=100)
+
+
+class Variance:
+	def time_variance(self):
+		try:
+			data = np.random.randn(10000, 10000)
+			import numba
+
+			@numba.njit(cache=True)
+			def stats_variance_1d(data, ddof=0):
+				a,b = 0,0
+				for i in data:
+					a = a + i
+					b = b + i * i
+				var = b / (len(data)) - ((a / (len(data))) ** 2)
+				var = var * (len(data) / (len(data) - ddof))
+				return var
+
+			@numba.njit(cache=True)
+			def stats_variance_2d(data, ddof=0, axis=1):
+				a, b = data.shape
+				if axis == 1:
+					var = np.zeros(a)
+					for i in range(a):
+						var[i] = stats_variance_1d(data[i], ddof=ddof)
+				else:
+					var = np.zeros(b)
+					for i in range(b):
+						var[i] = stats_variance_1d(data[:, i], ddof=ddof)
+				return var
+			return stats_variance_2d(data)
+		except ImportError:
+			data = np.random.randn(10000, 10000)
+			return np.var(data, axis=1)
+
+
+class CircStd:
+	def time_circ_std(self):
```

Comment:
Looks like its still tabs in here. I ask that this be converted to spaces

Reply:
Sure, look's like something went wrong with black.